package.json

```diff
@@ -27,9 +27,9 @@
   "devDependencies": {
     "@babel/preset-typescript": "7.12.13",
     "@codesandbox/sandpack-react": "0.13.15",
-    "@docusaurus/core": "2.0.0-rc.1",
-    "@docusaurus/module-type-aliases": "^2.0.0-rc.1",
-    "@docusaurus/preset-classic": "2.0.0-rc.1",
+    "@docusaurus/core": "2.0.1",
+    "@docusaurus/module-type-aliases": "2.0.1",
```

Comment:
What was the reason for this package to use caret dep? Should that stay that way?

Reply:
No reason, use exact versions